source/content/support.md

```diff
@@ -22,7 +22,7 @@ Pantheon offers a range of support packages that extend to include all the featu
 
 Keep these tips in mind when opening a new support ticket or chat, to help us more quickly resolve your issue:
 
- - **Open the Ticket in the Right Place:** Contacting Support from your User Dashboard will not provide the same support level as a Site Dashboard owned by an Organization with a higher support package.
+ - **Open the Ticket in the Right Place:** Contacting Support from your User Dashboard will not provide the same support level as a Site Dashboard owned by an Organization with a higher support package. For a visual, check out the [Support Tab from the Organization Dashboard](#the-support-tab-from-the-organization-dashboard).
```

Comment:
The previous sentence cites only the Site Dashboard. It should be edited to reflect site or org dash.